src/lib/Components/Consignments/Components/DoneButton.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react"
+
+import BottomAlignedButton from "./BottomAlignedButton"
+
+export interface DoneButtonProps {
+  onPress: () => void
+  children?: any[]
+}
+
+const render = (props: DoneButtonProps) => {
+  const doneButtonStyles = {
+    backgroundColor: "black",
+    marginBottom: 0,
+    paddingTop: 18,
+    height: 56,
+  }
```

Comment:
Please note that I removed at `marginBottom: 20` from these styles since it looked all funky. Let me know if it should still be there 👍